massgov/openmass
# DP-22403: Make existing paragraph types on org page available for sections

**Description:**
Added Featured Message, Featured Item Mosaic and Organization Grid to the Organization Section paragraph type.


**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22403


**To Test:**
- Login and go to /node/add/org_page.
- Click on the "Content" tab.
- Expand the "Section Content" field.
- [ ] Verify there are buttons for: Add Featured Mosaic, Add Featured Message, Add Organization Grid.
- Click on "Add Featured Mosaic".
- [ ] Verify there is no heading field in the form.
- Click on the "Add Featured Message".
- [ ] Verify there is no heading field in the form.


**Screenshots/GIFs:**

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
